backend/globaleaks/settings.py

```diff
@@ -66,7 +66,7 @@ def __init__(self):
         self.nodaemon = False
 
         # thread pool size of 1
-        self.orm_tp = ThreadPool(0, 1)
+        self.orm_tp = ThreadPool(1, 1)
```

Comment:
Thought you had forgotten this change here.

After looking at [the docs](http://twistedmatrix.com/documents/8.2.0/api/twisted.python.threadpool.ThreadPool.html), I see no reason to not keep it at `(1, 1)`